man/logind.conf.xml

```diff
@@ -232,17 +233,19 @@
         <literal>power-switch</literal> udev tag will be watched for
         key/lid switch events. <varname>HandlePowerKey=</varname>
         defaults to <literal>poweroff</literal>.
-        <varname>HandleSuspendKey=</varname> and
-        <varname>HandleLidSwitch=</varname> default to
+        <varname>HandleSuspendKey=</varname>,
+        <varname>HandleLidSwitch=</varname> and
+        <varname>HandleLidSwitchAC=</varname> default to
```

Comment:
The "AC" name is unfortunate, because not all supply is AC. At least it's possible to charge some smaller notebooks over USB. I wonder if we should just call this "HandleLidSwitchPower" instead. I see that this misnomer originates in the kernel, but we shouldn't propagate it.